digitalocean/cdn/resource_cdn_test.go

```diff
@@ -39,6 +39,33 @@ func TestAccDigitalOceanCDN_Create(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanCDN_CreateWithNeedsCloudflareCert(t *testing.T) {
+
+	bucketName := generateBucketName()
+	cdnCreateConfig := fmt.Sprintf(testAccCheckDigitalOceanCDNConfig_CreateWithNeedsCloudflareCert, bucketName)
+
+	expectedOrigin := bucketName + originSuffix
+	expectedTTL := "3600"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckDigitalOceanCDNDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: cdnCreateConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDigitalOceanCDNExists("digitalocean_cdn.foobar"),
+					resource.TestCheckResourceAttr(
+						"digitalocean_cdn.foobar", "origin", expectedOrigin),
+					resource.TestCheckResourceAttr("digitalocean_cdn.foobar", "ttl", expectedTTL),
+					resource.TestCheckResourceAttr("digitalocean_cdn.foobar", "certificate_name", "need-cloudflare-cert"),
```

Comment:
```suggestion
					resource.TestCheckResourceAttr("digitalocean_cdn.foobar", "certificate_name", "needs-cloudflare-cert"),
```

One tiny change needed here

Reply:
According to the const, it is `needs-cloudflare-cert`, right?

Reply:
oh yes 🤦‍♀️